src/styles/renderIndividual.module.css

```diff
@@ -18,7 +18,56 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  gap: 20px;
+  gap: 10px;
+}
+
```

Comment:
- [ ] As required for the project, your design to stick as much as possible to the one provided on [Zeplin](https://zeplin.io/) and All elements in design to be added. kindly ensure that you improve your `Header` styiling by adding the missing element, using the correct fonts(size, weight, family,...) on text, correct alignment, padding, margin, text color, Uppercase,... (Here sis the blue color to be used:` #0290ff`)
### Template design 
![Screenshot from 2023-05-19 18-32-47](https://github.com/Elhambasir/bookstore-add-reducers-and-actions/assets/83042742/ec8af863-f213-429b-9f11-2a3ebcd25a1c)
### Your design 
![Screenshot from 2023-05-19 18-33-23](https://github.com/Elhambasir/bookstore-add-reducers-and-actions/assets/83042742/df08d407-c8ca-464d-b835-a91d7b1dc350)

Reply:
I got it